viewing_party/party.py

```diff
@@ -1,23 +1,140 @@
+from statistics import mode
+
 # ------------- WAVE 1 --------------------
 
+
 def create_movie(title, genre, rating):
-    pass
+    if title == None or genre == None or rating == None:
+        return None
+        
+    movie_dict = {
+        "title" : title,
+        "genre" : genre,
+        "rating" : rating
+    }
+
+    return movie_dict

```

Comment:
If we want to get even dryer, let's combine lines 10 and 16

```suggestion
    return {
        "title" : title,
        "genre" : genre,
        "rating" : rating
    }
```